tests/test_training.py

```diff
@@ -20,9 +20,10 @@
 class DummyTrainingLoop(SLCWATrainingLoop):
     """A wrapper around SLCWATrainingLoop."""
 
-    def __init__(self, model: Model, sub_batch_size: int):
+    def __init__(self, model: Model, sub_batch_size: int, automatic_memory_optimization: bool = False):
         super().__init__(model=model, optimizer=optim.Adam(lr=1.0, params=model.parameters()))
         self.sub_batch_size = sub_batch_size
+        self.automatic_memory_optimization = automatic_memory_optimization
```

Comment:
should be passed via `super().__init__`

Reply:
Done in de0d8987908ce329a4a398474f84690dcf5d4f5d.